.github/workflows/ci.yaml

```diff
@@ -49,6 +49,42 @@ jobs:
                   if-no-files-found: error
                   retention-days: 7
 
+    build-linux-cmake:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Persist the cmake build
+              uses: actions/cache@v2
+              id: cmake_build_cache
+              with:
+                  path: ~/cmake_build
+                  key: cmake_build-${{ runner.os }}-${{ github.sha }}
```

Comment:
Nice! Are there any times when this build directory needs to be cleaned? I don't know much about cmake but I recall sometimes having to rm the build directory after changing the build config

Reply:
I reorganized the CI jobs and removed the cache, because I could not make it be shared between jobs. Now the building and testing are part of the same job. It may be due to the fact that the build dir is huge. On my system when building in release with debug info it is 70+ GiB.

Reply:
Sounds good to me. I use the [artifact action](https://github.com/facebookresearch/CompilerGym/blob/development/.github/workflows/ci.yaml#L47-L53) to share the built wheels between the various test jobs, but I guess a 70+ GB artifact wouldn't work :)